fedimint-testing/src/ln/mock.rs

```diff
@@ -163,4 +163,34 @@ impl ILnRpcClient for FakeLightningTest {
     ) -> Result<EmptyResponse, LightningRpcError> {
         Ok(EmptyResponse {})
     }
+
+    async fn connect_to_peer(
+        &self,
+        _pubkey: String,
+        _host: String,
+    ) -> Result<EmptyResponse, LightningRpcError> {
+        Ok(EmptyResponse {})
```

Comment:
`unimplemented!` is probably better for these functions, we shouldn't ever use these in the mocks

Reply:
Done.